hazelcast/src/main/java/com/hazelcast/dataconnection/impl/jdbcproperties/HikariTranslator.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright (c) 2008-2023, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.dataconnection.impl.jdbcproperties;
+
+import com.zaxxer.hikari.HikariConfig;
+import com.zaxxer.hikari.util.PropertyElf;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Properties;
+import java.util.Set;
+import java.util.concurrent.atomic.AtomicInteger;
+
+public class HikariTranslator {
+
+    private static final Map<String, String> PROPERTY_MAP = new HashMap<>();
+
+    private final AtomicInteger counter;
+    private final String name;
+
+    static {
+        PROPERTY_MAP.put(DataConnectionProperties.JDBC_URL, "jdbcUrl");
+        PROPERTY_MAP.put(DataConnectionProperties.USERNAME, "username");
+        PROPERTY_MAP.put(DataConnectionProperties.PASSWORD, "password");
```

Comment:
I thought we agreed that we will pass `user` and `password` as property to the jdbc driver, (via the `dataSource` prefix.